Extensions/popupLyrics.js

```diff
@@ -100,6 +100,8 @@ function PopupLyrics() {
 			const baseURL = `https://apic-desktop.musixmatch.com/ws/1.1/macro.subtitles.get?format=json&namespace=lyrics_synched&subtitle_format=mxm&app_id=web-desktop-app-v1.0&`;
 
 			const durr = info.duration / 1000;
+			const tokens = userConfigs.services.musixmatch.token.split("|");
+			const usertoken = tokens[Math.floor(Math.random() * tokens.length)];
```

Comment:
After discussing with team, this is unnecessary. Lyrics Plus doesn't need two or more tokens to work and doesn't make kinda sense because there is very high change it will pick a token that is "rate-limited" or "doesn't work". I also don't think someone would use it since it would be an undocumented feature. I will implement automatic token generation in a few days instead